diff --git a/third_party/blink/renderer/core/frame/navigator.h b/third_party/blink/renderer/core/frame/navigator.h
index 4572df017cf40ca2f2b8b746ca1aeea3021a277c..dac7921e04e0a1984f6c3bd32cb4aa3703bf293b 100644
--- a/third_party/blink/renderer/core/frame/navigator.h
+++ b/third_party/blink/renderer/core/frame/navigator.h
@@ -33,6 +33,7 @@
 #include "third_party/blink/renderer/platform/heap/handle.h"
 #include "third_party/blink/renderer/platform/supplementable.h"
 #include "third_party/blink/renderer/platform/wtf/forward.h"
+#include "brave/third_party/blink/renderer/core/frame/navigator_brave.h"
 
 namespace blink {
 
@@ -70,8 +71,9 @@ class CORE_EXPORT Navigator final : public ScriptWrappable,
   void SetUserAgentMetadataForTesting(UserAgentMetadata);
 
   void Trace(blink::Visitor*) override;
-
+  NavigatorBrave* brave() { if (!brave_) { brave_ = MakeGarbageCollected<NavigatorBrave>(); } return brave_; }
  private:
+  Member<NavigatorBrave> brave_;
   UserAgentMetadata metadata_;
 };
 
