diff --git a/third_party/blink/renderer/core/frame/navigator_user_agent.idl b/third_party/blink/renderer/core/frame/navigator_user_agent.idl
index a7bbe071b5d173243f7d19ba04190937c566c617..2ec4dd8f8c2a613e706ac1164429a52bc4d14723 100644
--- a/third_party/blink/renderer/core/frame/navigator_user_agent.idl
+++ b/third_party/blink/renderer/core/frame/navigator_user_agent.idl
@@ -5,7 +5,7 @@
 // https://github.com/WICG/ua-client-hints
 
 [
-    RuntimeEnabled=UserAgentClientHint,
+//  RuntimeEnabled=UserAgentClientHint,
     Exposed=Window
 ] interface mixin NavigatorUserAgent {
   [SecureContext, CallWith=ScriptState] Promise<UserAgent> getUserAgent();
